<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminScreen</class>
 <widget class="QDialog" name="AdminScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ADMIN PANEL</string>
  </property>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image:url(&quot;:/images/Back.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelAppName">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #4382EC;
	font: 900 11pt &quot;Roboto&quot;;
	border-radius: 10px;
	text-align: center;
	background-color: #E2EEFF;
}
	</string>
      </property>
      <property name="text">
       <string>             Super Market Management System </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>871</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget {
  background-color: #BBD6FD;
  border-radius: 10px;
} 

QTabBar::tab {
  background-color: #E2EEFF;
  color: #4382EC;
 }

 QTabBar::tab:selected {
 padding-top: 7px;
 padding-bottom: 7px;
  background: #BBD6FD;
  color: #4382EC;
 font-weight: bold;
	border-top: 2px solid #4382EC;
	border-left: 2px solid #4382EC;
	border-bottom: 2px solid #4382EC;
 }</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="shopData">
       <attribute name="title">
        <string>Shop</string>
       </attribute>
       <widget class="QTableView" name="tableViewShopData">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>461</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
color: black;
background-color: #E2EEFF;
padding: 5px;
}

QTableView QHeaderView {
    background-color: #E2EEFF;
	color: #4382EC;
	font: 900 11pt &quot;Roboto&quot;;
	font-weight: bold;
}

QTableView QHeaderView::section {
	border: 2px solid #4382EC;
	border-radius: 5px;
    background-color: #E2EEFF;
	color: #4382EC;
	font: 500 11pt &quot;Roboto&quot;;
	font-weight: bold;
}

QTableView QHeaderView::section:checked {
    background-color: #BBD6FD;
    color: #4382EC;
    border-radius: 10px;
	font: 900 11pt &quot;Roboto&quot;;
	font-weight: bold;
}

QScrollBar {
    background-color: #4382EC;
    border-radius: 10px;
    margin-right: 5px;
	margin-left: 5px;
}

QTableView::item {
    border-radius: 10px;
	border: 1px solid  #4382EC;
}

QTableView::item:selected {
    background-color: #4382EC;
    color: white;
	font-weight: bold;
}
</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="employeeData">
       <attribute name="title">
        <string>Employee</string>
       </attribute>
       <widget class="QTableView" name="tableViewEmployeeData">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>461</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
color: black;
background-color: #E2EEFF;
padding: 5px;
}

QTableView QHeaderView {
    background-color: #E2EEFF;
	color: #4382EC;
	font: 900 11pt &quot;Roboto&quot;;
	font-weight: bold;
}

QTableView QHeaderView::section {
	border: 2px solid #4382EC;
	border-radius: 5px;
    background-color: #E2EEFF;
	color: #4382EC;
	font: 500 11pt &quot;Roboto&quot;;
	font-weight: bold;
}

QTableView QHeaderView::section:checked {
    background-color: #BBD6FD;
    color: #4382EC;
    border-radius: 10px;
	font: 900 11pt &quot;Roboto&quot;;
	font-weight: bold;
}

QScrollBar {
    background-color: #4382EC;
    border-radius: 10px;
    margin-right: 5px;
	margin-left: 5px;
}

QTableView::item {
    border-radius: 10px;
	border: 1px solid  #4382EC;
}

QTableView::item:selected {
    background-color: #4382EC;
    color: white;
	font-weight: bold;
}
</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="LogOut">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image:url(&quot;:/images/LogOut.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
border: 0;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonAddData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image:url(&quot;:/images/Insert.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRemoveData">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image:url(&quot;:/images/remove.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUpdateData">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image:url(&quot;:/images/Update.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonChangePassword">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>620</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image:url(&quot;:/images/Security.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="searchFrame">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame #searchFrame {
	color: white;
	background-color: #4382EC;
	border: none;
	border-radius: 10px;
	text-align: center;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color : #BBD6FD;
color: #4382EC;
border: 2px solid #4382EC;
font-weight: bold;
}

QLineEdit:hover
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
font-weight: bolder;
}

QLineEdit:focus
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
color: #4382EC;
font-weight: italic;
}</string>
    </property>
    <property name="placeholderText">
     <string>Search Shop ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #4382EC;
	font-weight: bold;
	border: 2px solid #4382EC;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
background-color: #3569B2;
border: 3px solid #BBD6FD;
}
	</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearFilter">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>630</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #4382EC;
	font-weight: bold;
	border: 2px solid #4382EC;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
background-color: #3569B2;
border: 3px solid #BBD6FD;
}
	</string>
   </property>
   <property name="text">
    <string>Clear Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label
{
color: #BBD6FD;
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>670</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: #E2EEFF;
color: #4382EC;
border-radius: 10px;
text-align: center;

}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
